docs-ref-autogen/TOC.md

```diff
@@ -12,6 +12,7 @@
 ## [Output formats](../docs-ref-conceptual/format-output-azure-cli.md)
 ## [Query](../docs-ref-conceptual/query-azure-cli.md)
 ## [Subscriptions](../docs-ref-conceptual/manage-azure-subscriptions-azure-cli.md)
+## [Clouds](../docs-ref-conceptual/manage-cloud-sets-azure-cli.md)

```

Comment:
Does the URL have to change? It still mentions cloud sets. If it's not user-facing then it's less of an issue.